pkg/model/hvs/trust_report.go

```diff
@@ -43,48 +43,40 @@ type MismatchField struct {
 	Exclude_Tags             []string               `json:"excluding_tag,omitempty"`
 	ExpectedTag              []byte                 `json:"expected_tag,omitempty"`
 	Tags                     map[string]string      `json:"tags,omitempty"`
 }
 
 type Fault struct {
-	Name                    string                   `json:"fault_name"`
-	Description             string                   `json:"description"`
-	PcrIndex                *types.PcrIndex          `json:"pcr_index,omitempty"`
-	PcrBank                 *types.SHAAlgorithm      `json:"pcr_bank,omitempty"`
-	ExpectedPcrValue        *string                  `json:"expected_pcrvalue,omitempty"`
-	ActualPcrValue          *string                  `json:"actual_pcrvalue,omitempty"`
-	MissingEntries          []types.EventLog         `json:"missing_entries,omitempty"`
-	MissingEventEntries     []types.EventLogCriteria `json:"missing_event_entries,omitempty"`
-	UnexpectedEntries       []types.EventLog         `json:"unexpected_entries,omitempty"`
-	UnexpectedEventEntries  []types.EventLogCriteria `json:"unexpected_event_entries,omitempty"`
-	ActualEventLog          []types.EventLog         `json:"actual_events,omitempty"`
-	ExpectedEventLog        []types.EventLog         `json:"expected_events,omitempty"`
-	IncludesEventLogEntries []types.EventLog         `json:"includes_entries,omitempty"`
-	ExcludeTags             []string                 `json:"exclude_tags,omitempty"`
-	FlavorId                *uuid.UUID               `json:"flavor_id,omitempty"`
-	UnexpectedMeasurements  []ta.FlavorMeasurement   `json:"unexpected_measurements,omitempty"`
-	MissingMeasurements     []ta.FlavorMeasurement   `json:"missing_measurements,omitempty"`
-	MismatchMeasurements    []ta.FlavorMeasurement   `json:"mismatch_measurements,omitempty"`
-	ExpectedValue           *string                  `json:"expected_value,omitempty"`
-	CalculatedValue         *string                  `json:"calculated_value,omitempty"`
-	ActualValue             *string                  `json:"actual_value,omitempty"`
-	MeasurementId           *string                  `json:"measurement_id,omitempty"`
-	FlavorDigestAlg         *string                  `json:"flavor_digest_alg,omitempty"`
-	MeasurementDigestAlg    *string                  `json:"measurement_digest_alg,omitempty"`
+	Name                   string                   `json:"fault_name"`
+	Description            string                   `json:"description"`
+	PcrIndex               *types.PcrIndex          `json:"pcr_index,omitempty"`
+	PcrBank                *types.SHAAlgorithm      `json:"pcr_bank,omitempty"`
+	ExpectedPcrValue       *string                  `json:"expected_pcrvalue,omitempty"`
+	ActualPcrValue         *string                  `json:"actual_pcrvalue,omitempty"`
+	MissingEntries         []types.EventLogCriteria `json:"missing_entries,omitempty"`
+	UnexpectedEntries      []types.EventLogCriteria `json:"unexpected_entries,omitempty"`
```

Comment:
Why are there missing fields in this structure?

Reply:
Missing entry field is used in Pcr Event Log Missing Fault structure for the detailed information in report.
Missing entries - when subtracting actual entries from expected entries, if anything left in unexpected entries, called missing entries.

Reply:
One of the design goals was to not alter the output of the reports.  Will other components that consume reports (like iHub) break if these fields are removed?

Reply:
Earlier, I misunderstood the above question and wrongly explained about missing entries. Please find the explanation below:

The fields that are removed will not alter the output of the trust report since these fields(**ActualEventLog,ExpectedEventLog,IncludesEventLogEntries**) are not used in fault structure and not in any other verification code(also not present in labelled version ,only introduced these fields during flavor verification code development). 

And,Previously, **MissingEntries,UnexpectedEntries** fields were used in ESXI trust report and  **MissingEventEntries,UnexpectedEventEntries** fields were used in the Linux trust report. Now, We have changed the ESXI flow to dynamic verification. So ,we can keep the same MissingEntries ,UnexpectedEntries fields to both the flow.

Reply:
I will ask Tim, but I'm pretty sure what his answer will be: "we can't break compatibility with the report format".